src/bootstrap.py

```
@@ -184,4 +208,49 @@ def build_collection_pipeline():
+    return pipeline
+
+
+# ---------------------------------------------------------------------------
+# Translation Pipeline：翻譯流程的依賴組裝
+# ---------------------------------------------------------------------------
+
+def build_translation_pipeline():
+    """
+    組裝翻譯 pipeline。
+
+    回傳翻譯相關服務，可用於定時翻譯任務。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)